designer/server/src/views/400.njk

```diff
@@ -0,0 +1,11 @@
+{% extends 'layouts/page.njk' %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+
+{% block content %}
+  {% call appPageBody({
+    heading: pageHeading
+  }) %}
+    <p class="govuk-body">The service couldn't process your request.</p>
```

Comment:
What does "process your request" mean to a user?

Watch out for the [negative contraction](https://www.gov.uk/guidance/style-guide/a-to-z#contractions) "couldn't" too

```suggestion
    <p class="govuk-body">The service could not process your request.</p>
```

Reply:
I've put the question to Dan, will look to merge this unless they weigh in and let them improve it later.